extensions/amp-sidebar/validator-amp-sidebar.protoascii

```diff
@@ -89,3 +95,38 @@ tags: {  # amp-sidebar > nav
     child_tag_name_oneof: "UL"
   }
 }
+
+# Reference Points apply only to direct children, and all direct children must
```

Comment:
@honeybadgerdontcare do you see any risk (thinking weird edge case bugs) with this that may result in invalidating existing cases? I tested for 4-5 live pages and seemed fine, existing tests passed to.

Reply:
Thanks. Added `AMP4EMAIL` to both